styles/mystyles.css

```
@@ -22,20 +24,74 @@
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
-:root {
+/*Paars #5722CD;
+Blauw #4845E4;
+Roze #ED4385;
+Zwart #0C0C0C (voor standaard tekst);
+Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen:
+Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)
+Grijs #7A7A7A (voor de service-tegel tekst);*/
 
+:root {
+    --font-family-heading: 'Merriweather', serif;
+    --font-family-text: 'Roboto', sans-serif;
 
+    --paars: #5722CD;
+    --blauw: #4845E4;
+    --roze: #ED4385;
+    --zwart: #0C0C0C;
+    --lichtpaars: #F2F0FF;
+    --lichtgrijs: #F3F6F9;
+    --grijs: #7A7A7A;
 }
 
 body{
+    height: 100vh;
+    font-family: 'Roboto', 'Merriweather', 'Arial', sans-serif;
+    font-weight: 300;
+    font-size: 16px;
+}
+
 
+img{
+    width: 100%;
 }
```

Hier zou ik dan de height specificeren, maar dan wel de max-height, omdat hij anders altijd die height zou aannemen:
```suggestion
    max-height: 400px;
}
```